Mathlib/Topology/Basic.lean

```diff
@@ -848,7 +848,7 @@ scoped[Topology] notation "𝓝" => nhds
 scoped[Topology] notation "𝓝[" s "] " x:100 => nhdsWithin x s
 
 /-- Notation for the filter of punctured neighborhoods of a point. -/
-scoped[Topology] notation "𝓝[≠] " x:100 => nhdsWithin x {x}ᶜ
+scoped[Topology] notation "𝓝[≠] " x:100 => nhdsWithin x (@singleton _ (Set _) instSingletonSet x)ᶜ
```

Comment:
Does the following work?
```suggestion
scoped[Topology] notation "𝓝[≠] " x:100 => nhdsWithin x ({x}ᶜ : Set _)
```

Reply:
Eric, you can see from the git history that your proposition was my first "fix", but it didn't work.

Reply:
I'm seeing `nhdsWithin x ({x} : Set _)ᶜ` in the git history -- Eric's suggesting `nhdsWithin x ({x}ᶜ : Set _)`

Reply:
I've tried a few things and I can't get it to work like that.

I did find this workaround though:
```
set_option quotPrecheck false in
/-- Notation for the filter of punctured neighborhoods of a point. -/
scoped[Topology] notation "𝓝[≠] " x:100 => letI z := x; nhdsWithin z ({z}ᶜ)
```
I'm not sure what's going on or why this helps at this point.